sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -112,6 +169,23 @@ public void info(String message) {
         }
     }
 
+    /**
+     * Logs a message at {@code info} log level.
+     *
+     * <p><strong>Code samples</strong></p>
+     *
+     * <p>Logging a message at information log level.</p>
+     *
+     * {@codesnippet com.azure.core.util.logging.clientlogger.deferredInfo#supplier}
+     *
+     * @param message a supplier for for logging messages.
+     */
+    public void deferredInfo(Supplier<String> message) {
```

Comment:
These `deferred*` methods can be dropped in favor of `log(LogLevel, Supplier<>)` method.

Reply:
Fixed